distributions/Dist_rdr335.py

```diff
@@ -1,20 +1,21 @@
 import numpy as np
 from base_distribution import BaseDistribution
 
+c = np.sqrt(2/np.pi)
```

Comment:
defining a constant outside of the code will cause problems.
You can add `self.c = np.sqrt(2/np.pi)` in the `__init__` method similar to `self._x_max`